packages/react-components/src/components/Search/Search.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react';
+import cx from 'clsx';
+import {
+  Search as SearchIcon,
+  Close,
+} from '@livechat/design-system-icons/react/material';
+import { Icon } from '../Icon';
+import { Loader } from '../Loader';
+
+import styles from './Search.module.scss';
+import { KeyCodes } from '../../utils/keyCodes';
+
+const baseClass = 'search-input';
+const inputBaseClass = `${baseClass}__input`;
+
+export const enum SearchSize {
+  Compact = 'compact',
+  Medium = 'medium',
+  Large = 'large',
+}
+
+export interface ISearchProps {
+  isCollapsable?: boolean;
+  isDisabled?: boolean;
+  isLoading?: boolean;
+  placeholder?: string;
+  size?: SearchSize;
+  value?: string;
+  onChange: (value: string) => void;
+}
+
+export const Search: React.FC<ISearchProps> = ({
```

Comment:
```suggestion
export const SearchInput: React.FC<ISearchInputProps> = ({
```

AFAIR Design System project don't use "I" prefix for interfaces but I may be not familiar with the current convention.

Reply:
@AlithAnar looking at other examples it looks like on `v1` this prefix is used

Reply:
👍